### PR TITLE
[docs] update documentation for connect-service-upstreams

### PR DESCRIPTION
**Pull request by s-christoff**
Adds documentation for services, multiple services, and prepared
query under connect-service-upstreams annotation.

### CODE REVIEW

File: website/source/docs/platform/k8s/connect.html.md [hunk 1]
```diff
@@ -174,10 +174,28 @@ Annotations can be used to configure the injection behavior.
...
   services that this pod needs to connect to via Connect along with a static
-  local port to listen for those connections. Example: `db:1234,auth:6789`
-  will start two local listeners for `db` on port 1234 and `auth` on port
-  6789, respectively. The name of the service is the name of the service
-  registered with Consul. This value defaults to no upstreams.
+  local port to listen for those connections. 
+
+    * Services
+   
+        The name of the service is the name of the service registered with Consul. Datacenters can optionally be specified within this annotation.
```
Comment (judithpatudith) [inline website/source/docs/platform/k8s/connect.html.md]: Put the second sentence into active voice: You can optionally specify datacenters with this annotation.

File: website/source/docs/platform/k8s/connect.html.md [hunk 2]
```diff
@@ -174,10 +174,28 @@ Annotations can be used to configure the injection behavior.
...
+    * Services
+   
+        The name of the service is the name of the service registered with Consul. Datacenters can optionally be specified within this annotation. 
+        ```annotations:
+        "consul.hashicorp.com/connect-service-upstreams":"[service-name]:[port]:[optional datacenter]"
+        ```
+
+    * Multiple Services
+
+        If multiple services would like to be specified they are delimited by commas.
```
Comment (judithpatudith) [inline website/source/docs/platform/k8s/connect.html.md]: Active voice too: If you would like to specify multiple services, delimit them with commas.

File: website/source/docs/platform/k8s/connect.html.md (hunk 1 above)
Comment (s-christoff) [reply website/source/docs/platform/k8s/connect.html.md]: Heya, thank you so much for this!

File: website/source/docs/platform/k8s/connect.html.md (hunk 2 above)
Comment (s-christoff) [reply website/source/docs/platform/k8s/connect.html.md]: Thank you, should be resolved :)

File: website/source/docs/platform/k8s/connect.html.md [hunk 3]
```diff
@@ -174,10 +174,30 @@ Annotations can be used to configure the injection behavior.
...
-  local port to listen for those connections. Example: `db:1234,auth:6789`
-  will start two local listeners for `db` on port 1234 and `auth` on port
-  6789, respectively. The name of the service is the name of the service
-  registered with Consul. This value defaults to no upstreams.
+  local port to listen for those connections. 
+
+    * Services
+   
+        The name of the service is the name of the service registered with Consul. You can optionally specify datacenters with this annotation. 
+        ```annotations:
```
Comment (judithpatudith) [inline website/source/docs/platform/k8s/connect.html.md]: I think we still need an extra line between this code block and the preceding paragraph. The other fixes look good!

File: website/source/docs/platform/k8s/connect.html.md [hunk 4]
```diff
@@ -174,10 +174,37 @@ Annotations can be used to configure the injection behavior.
...
+
+    * [Prepared Query](https://www.consul.io/docs/connect/proxies.html#upstreams)
+
+        ```annotations:
+        "consul.hashicorp.com/connect-service-upstreams": "prepared_query:[query name]:[port]"
+        ````
+
+    * Multiple Upstreams
+
+      If you would like to specify multiple services or upstreams, delimit them with commas
```
Comment (judithpatudith) [inline website/source/docs/platform/k8s/connect.html.md]:
```suggestion
        If you would like to specify multiple services or upstreams, delimit them with commas
```

Comment (judithpatudith) [reply website/source/docs/platform/k8s/connect.html.md]: This fixes the indentation on the multiple upstreams bullet.

**Pull request merged**